near-contract-standards/src/event.ts

```diff
@@ -0,0 +1,19 @@
+import { near } from "near-sdk-js";
+
+export class NearEvent {
```

Comment:
Is this class intended to be instantiated? Should it be abstract?

Reply:
Yes, it's never instantiated, making it an abstract class